spot-electron/config.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+
+    /**
+     * The default URL to connect to.
+     */
+    defaultSpotURL: 'https://spot.8x8.vc/spot',
```

Comment:
Maybe add a process.env variable right now to make this configurable immediately? That would be my preference, but not a requirement.

Reply:
Good call, thanks!